includes/abstracts/abstract-wc-stripe-payment-gateway.php

```diff
@@ -1162,7 +1162,7 @@ public function update_fees( $order, $balance_transaction_id ) {
 				}
 			}
 		} else {
-			WC_Stripe_Logger::log( 'Unable to update fees/net meta for order: ' . $order->get_id() );
+			WC_Stripe_Logger::warning( 'Unable to update fees/net meta for order: ' . $order->get_id() );
```

Comment:
Similar to my previous comment, is it worth adding the error details to the context?
```suggestion
			WC_Stripe_Logger::warning( 'Unable to update fees/net meta for order: ' . $order->get_id(), [ 'error' => $balance_transaction->error ] );
```